privaterelay/settings.py

```diff
@@ -435,6 +437,273 @@ def _get_initial_middleware() -> list[str]:
+        },
+    },
+    "usd": {
+        "en": {
+            "monthly": {
+                "id": "price_1LXUcnJNcmPzuWtRpbNOajYS",
+                "price": "$?.??",
+            },
+            "yearly": {
+                "id": "price_1LXUdlJNcmPzuWtRKTYg7mpZ",
+                "price": "$?.??",
+            },
+        },
+        "gb": {
+            "monthly": {
+                "id": "price_1LYCHpJNcmPzuWtRhrhSYOKB",
+                "price": "$?.??",
+            },
+            "yearly": {
+                "id": "price_1LYCIlJNcmPzuWtRQtYLA92j",
+                "price": "$?.??",
+            },
+        },
+    },
+}
+PERIODICAL_PREMIUM_PLAN_COUNTRY_LANG_MAPPING = {
+    # Austria
+    "at": {"de": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["de"]},
+    # Belgium
+    "be": {
+        "fr": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["fr"],
+        "de": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["de"],
+        "nl": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["nl"],
+    },
+    # Switzerland
+    "ch": {
+        "fr": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["chf"]["fr"],
+        "de": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["chf"]["de"],
+        "it": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["chf"]["it"],
+    },
+    # Germany
+    "de": {
+        "de": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["de"],
+    },
+    # Spain
+    "es": {
+        "es": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["es"],
+    },
+    # France
+    "fr": {
+        "fr": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["fr"],
+    },
+    # Ireland
+    "ie": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["en"],
+    },
+    # Italy
+    "it": {
+        "it": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["it"],
+    },
+    # Netherlands
+    "nl": {
+        "nl": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["nl"],
+    },
+    # Sweden
+    "se": {
+        "sv": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["sv"],
+    },
+    # Finland
+    "fi": {
+        "fi": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["euro"]["fi"],
+    },
+    "us": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["en"],
+    },
+    "gb": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    "ca": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    "nz": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    "my": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    "sg": {
+        "en": PERIODICAL_PREMIUM_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+}
+
+PHONE_PLAN_ID_MATRIX = {
+    "usd": {
+        "en": {
+            "monthly": {
+                "id": "price_1LXUenJNcmPzuWtRw3rhjQNP",
+                "price": "$?.??",
+            },
+            "yearly": {
+                "id": "price_1LXUhcJNcmPzuWtRHUFHVk12",
+                "price": "$?.??",
+            },
+        },
+    },
+}
+PHONE_PLAN_COUNTRY_LANG_MAPPING = {
+    "us": {
+        "en": PHONE_PLAN_ID_MATRIX["usd"]["en"],
+    },
```

Comment:
suggestion (blocking): I think `ca` needs to be in here so when we detect someone coming from `ca` country they can see the phone UI.
```suggestion
    "us": {
        "en": PHONE_PLAN_ID_MATRIX["usd"]["en"],
    },
    "ca": {
        "en": PHONE_PLAN_ID_MATRIX["usd"]["en"],
    },
```

Reply:
Same for Austria and Ireland, but not plan IDs were known for those yet. Once we get one for Canada we can add it in here. (And by leaving it out, it's obvious when QA connects from Canada that we haven't received it yet.)

Reply:
OK, what I've done for now is leave commented-out traces in the code that indicate that we still need values there, without risking going to prod with invalid plan IDs: 6b2f3e84

Reply:
We'll have to ask PROD to be sure, but I think the ca plan will be the same as us?

Reply:
Resolving this convo with the TODOs in the code. Thanks!